Source/Hurl.RulesetManager/MainWindow.xaml

```diff
@@ -0,0 +1,119 @@
+﻿<ui:FluentWindow
+    x:Class="Hurl.RulesetManager.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:Hurl.RulesetManager"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
+    Title="MainWindow"
+    Width="800"
+    Height="450"
+    ExtendsContentIntoTitleBar="True"
+    WindowBackdropType="Mica"
+    WindowCornerPreference="Round"
+    WindowStartupLocation="CenterScreen"
+    mc:Ignorable="d">
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="1*" />
+            <ColumnDefinition Width="4*" />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="10*" />
+        </Grid.RowDefinitions>
+
+        <ui:TitleBar
+            Title="Ruleset Manager - Hurl"
+            Grid.ColumnSpan="2"
+            Icon="pack://application:,,,/Assets/applicationIcon-256.png" />
+
+        <StackPanel Grid.Row="1" Grid.Column="0">
+            <TextBlock FontSize="18" Text="Rulesets:" />
+            <ui:Button HorizontalAlignment="Stretch">Create</ui:Button>
+            <ui:Card VerticalAlignment="Stretch">
+                <StackPanel />
+            </ui:Card>
+        </StackPanel>
+
+        <StackPanel Grid.Row="1" Grid.Column="1">
+            <TextBlock
+                x:Name="WarnText"
+                Height="0"
+                HorizontalAlignment="Center"
+                Foreground="Red"
+                Text="Atleast input Target Browser and Rules fields!"
+                Visibility="Hidden" />
+
+            <ui:Card Padding="10">
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="1*" />
```

Comment:
A `*` is the default, so you could also use `<RowDefinition />`

Reply:
Thank you, didn't know this....